dallinger/recruiters.py

```diff
@@ -604,6 +614,24 @@ def get_qualifications(self, hit_id, sandbox):
         }
 
 
+class DevProlificRecruiter(ProlificRecruiter):
```

Comment:
I propose that if `recruiter = prolific` in config.txt then running `dallinger debug` should now select `DevProlificRecruiter`, rather than `HotAirRecruiter`, as the recruiter (so we don't have to set `debug_recruiter` manually). Does that sound good to you?

Reply:
Done.